.github/workflows/scala3-build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Set up JDK 11
         uses: olafurpg/setup-scala@v10
         with:
-          java-version: adopt@1.11.0-9
+          java-version: adopt@1.11
```

Comment:
Let this run a bit more free (instead of sticking to `-9`.

Reply:
I like specific versions, but this is :+1: too :)

Reply:
I'm fine hardcoding to 1.11.0-11, it is unlikely to change in a while. My point was that 1.11.0-9 is not the latest LTS.